client/react-client-app/src/components/Error.jsx

```
@@ -0,0 +1,20 @@
+function Error({ children }) {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)